qemu/tests/enable_scatter_windows.py

```diff
@@ -39,41 +39,45 @@ def _is_process_finished(session, process_name):
             return False
         return process_name not in output
 
-    def _start_wireshark_session():
+    def _start_windump_session():
         """
-        Start a wireshark session and log network traffic to a file
+        Start a WinDump session and log network traffic to a file
         """
-        error_context.context("Start wireshark session", test.log.info)
+        error_context.context("Start WinDump session", test.log.info)
         session_serial = vm.wait_for_serial_login(timeout=timeout)
         guest_ip = vm.get_address()
         try:
-            run_wireshark_cmd = run_wireshark_temp % (host_ip, guest_ip)
+            run_windump_cmd = run_windump_temp % (host_ip, guest_ip)
             status, output = session_serial.cmd_status_output(
-                run_wireshark_cmd, timeout=timeout
+                run_windump_cmd, timeout=timeout
             )
-
             if status:
                 test.error(
-                    "Failed to start wireshark session, "
-                    "status=%s, output=%s" % (status, output)
+                    "Failed to start WinDump session, status=%s, output=%s"
+                    % (status, output)
                 )
             is_started = utils_misc.wait_for(
-                lambda: not _is_process_finished(session_serial, "tshark.exe"), 20, 5, 1
+                lambda: not _is_process_finished(
+                    session_serial, params.get("windump_name")
+                ),
+                20,
+                5,
+                1,
```

Comment:
Do we have to change it this way?

Reply:
Yes, The pre-commit tells me I must modify the code this way.